vote/vote.go

```diff
@@ -1069,10 +1184,51 @@ func Preload(ctx context.Context, flow flow.Getter, pollID int, meetingID int) e
 	q := ds.Poll(pollID)
 	q = q.Preload(q.EntitledGroupList().MeetingUserList().User())
 	q = q.Preload(q.EntitledGroupList().MeetingUserList().VoteDelegatedTo().User())
-	_, err := q.First(ctx)
+	poll, err := q.First(ctx)
 	if err != nil {
 		return fmt.Errorf("fetch preload data: %w", err)
 	}
+
```

Comment:
@bastianjoel Is there a better way in dsmodels to fetch the contend of a generic-relation field?

Reply:
No, it was easier to just implement helpers in the projector service directly as most of the time just one field is needed. 

https://github.com/OpenSlides/openslides-projector-service/blob/main/pkg/viewmodels/content_object.go